### DISCUSSION TITLE
Error when accessing graphql localhost

### DISCUSSION THREAD

**Discussion by Isaac-Tait** [tinacms/tinacms#4190] [Help] [answered]
When I run `yarn dev` it spins up a Tina Config API URL at localhost:4001/graphql. However, when I navigate to the URL I get this error: 

`{"errors":[{}]}`

My console shows: `Error: Body must be a string. Received: undefined.`

Any ideas on what is going on?

**Comment by kldavis4** [comment]
localhost:4001 is the graphql API. The UI is at localhost:3000

**Reply by Isaac-Tait** [reply]
I am trying to access the graphql API when I get the errors. 

Taking a look at the [Graphql](https://tina.io/docs/graphql/cli/) docs if I navigate to `http://localhost:4001/altair` I get numerous "calls" to admin (this is what my URL looks like after navigating to altair `http://localhost:4001/admin/altair/#/~/admin/#/~/admin/#/~/admin/#/~/admin/#/~/admin/#/~/admin/#/~/admin/#/~/admin/#/~/admin/#/~/admin/`)which, opens numerous sub-windows before I get this error: `Attempt to use history.replaceState() more than 100 times per 30 seconds`

**Reply by jamespohalloran** [reply, marked as answer]
That doc should be updated, try: http://localhost:3000/admin/index.html#/graphql
